### PR TITLE
Chapter 5: Filling in the layout

### PR DESCRIPTION
**Pull request by TuTranVan**
ready

### CODE REVIEW

**Comment by TuTranVan** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ end
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails"
 gem "rails", "~> 5.1.6", ">= 5.1.6.1"
+gem 'bootstrap-sass', '3.3.7'
```
Comment (phamvanbk11) [inline Gemfile]: nháy " nhé

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+<div class="center jumbotron">
+  <h1><%= t "welcome" %></h1>
+  <h2>
+    <%= t(".home_html", href1: link_to(t(".rort"), t(".url_rails"))) %>
+  </h2>
+  <%= link_to t("static_pages.home.signup"), signup_path, class: "btn btn-lg btn-primary" %>
+</div>
+
+<%= link_to image_tag("rails.png", alt: "Rails logo"),
+            'http://rubyonrails.org/' %>
```
Comment (phamvanbk11) [inline app/views/static_pages/home.html.erb]: nháy "

**Comment by phamvanbk11** [comment]
commented

**Comment by TuTranVan** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**